doc/classes/AnimationMixer.xml

```diff
@@ -135,6 +135,19 @@
 				    move_and_slide()
 				[/gdscript]
 				[/codeblocks]
+				If [member root_motion_local] is [code]true[/code], return the pre-multiplied translation value with the inverted rotation.
+				In this case, the code can be written as follows:
+				[codeblocks]
+				[gdscript]
+				func _process(delta):
+				    if Input.is_action_just_pressed("animate"):
+				        state_machine.travel("Animate")
+				    set_quaternion(get_quaternion() * animation_tree.get_root_motion_rotation())
+				    var velocity: Vector3 = get_quaternion() * animation_tree.get_root_motion_position() / delta
```

Comment:
```suggestion
				    var velocity = get_quaternion() * animation_tree.get_root_motion_position() / delta
```
[Avoid](https://docs.godotengine.org/en/stable/contributing/documentation/docs_writing_guidelines.html#dynamic-vs-static-typing) static typing in the documentation